### ISSUE TITLE
cy.exec() fails when running Cypress in docker

### ISSUE THREAD

**Issue by andreaazzara** [cypress-io/cypress#517]
* Operating System: Mac OS Sierra 10.12.4
* Cypress Version: 0.19.2
* Browser/Browser Version: 

### Are you requesting a feature or reporting a bug?
Bug

### Current behavior:
In a Docker environment when running `cy.exec('echo test')` with a multi word command the following error is produced

```
> "spawn echo test ENOENT"
    at Object.cypressErr (http://localhost:44506/__cypress/static/js/cypress.js:5723:15)
    at Object.throwErr (http://localhost:44506/__cypress/static/js/cypress.js:5688:22)
    at Object.throwErrByPath (http://localhost:44506/__cypress/static/js/cypress.js:5715:21)
    at http://localhost:44506/__cypress/static/js/cypress.js:10251:33
    at tryCatcher (http://localhost:44506/__cypress/static/js/vendor.js:46486:31)
    at Promise._settlePromiseFromHandler (http://localhost:44506/__cypress/static/js/vendor.js:44621:31)
    at Promise._settlePromiseAt (http://localhost:44506/__cypress/static/js/vendor.js:44697:18)
    at Promise._settlePromises (http://localhost:44506/__cypress/static/js/vendor.js:44813:14)
    at Async._drainQueue (http://localhost:44506/__cypress/static/js/vendor.js:42180:16)
    at Async._drainQueues (http://localhost:44506/__cypress/static/js/vendor.js:42190:10)
    at MutationObserver.Async.drainQueues (http://localhost:44506/__cypress/static/js/vendor.js:42018:14)]]></failure>
    </testcase>
  </testsuite>
</testsuites>
```

while `cy.exec('echo')` or `cy.exec('ls')` do not fail.

**Comment by bahmutov** [comment]
Perfect, @andreaazzara thank you so much for taking time to report this. We will address this.

**Comment by brian-mann** [comment]
Funny you should bring this up. Internally we switched all of our CI containers to docker and I actually fixed this bug yesterday.

You'll see it go out in the next release `0.20.0`.

**Comment by RandallKent** [comment]
The code for this is done, but this has yet to be released. We'll update the issue and reference the changelog when it's released.

**Comment by brian-mann** [comment]
Fixed in [`0.20.0`](https://docs.cypress.io/guides/references/changelog.html#0-20-0)
